contribs/ev/src/main/java/org/matsim/contrib/ev/discharging/DischargingModule.java

```diff
@@ -41,6 +41,7 @@ public void install() {
 			protected void configureQSim() {
 				this.bind(DriveDischargingHandler.class).in( Singleton.class );
 				addMobsimScopeEventHandlerBinding().to(DriveDischargingHandler.class);
+				addMobsimListenerBinding().to(DriveDischargingHandler.class);
```

Comment:
This line should be replaced with:
`this.addQSimComponentBinding(EvModule.EV_COMPONENT).to(DriveDischargingHandler.class);`

Right now I doubt there is any event process. Maybe this is why so may tests failed.